cl/settings/django.py

```diff
@@ -189,6 +190,8 @@
     INSTALLED_APPS.append("django_extensions")
     MIDDLEWARE.append("debug_toolbar.middleware.DebugToolbarMiddleware")
 
+ASGI_APPLICATION = "cl.asgi.application"
```

Comment:
This is what loads `asgi.py` for development which needs to be part of the project namespace.